.github/ISSUE_TEMPLATE/localization_suggestion.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Localization Suggestion
+about: Report a problem or suggested change to Calculator's localized content.
+title: '[Localization] '
+labels: 'Area: World-Readiness'
+assignees: ''
+---
+<!--
+Please note:  We cannot _accept_ any change to our localized resources files. These files are automatically generated from an internal localization process.
+
+Before filing a bug
+- Ensure the bug reproduces on the latest version of the app.
+- Search existing issues and make sure this issue is not already filed.
+-->
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**Steps To Reproduce**
+<!--
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+-->
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Screenshots**
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Device and Application Information**
+ - OS Build:
+ - Architecture:
+ - Application Version:
+ - Region: 
+ - Dev Version Installed: 
+ 
+<!--
+Run the following commands in Powershell and copy/paste the output.
+" - OS Build: $([Environment]::OSVersion.Version)"
+" - Architecture: $((Get-AppxPackage -Name Microsoft.WindowsCalculator).Architecture)"
+" - Application Version: $((Get-AppxPackage -Name Microsoft.WindowsCalculator).Version)"
+" - Region: $((Get-Culture).Name)"
+" - Dev Version Installed: $(!((Get-AppxPackage -Name Microsoft.WindowsCalculator.Dev) -eq $null))"
```

Comment:
Recommended PowerShell code has `$null` always on the left hand side of comparisons.  Additionally, it's simplified if you use `-ne` instead of negating your expression and using `-eq`.

```suggestion
" - Dev Version Installed: $($null -ne (Get-AppxPackage -Name Microsoft.WindowsCalculator.Dev))"
```